### PR TITLE
Reduce verbosity, hiding some empty fields

### PR DESCRIPTION
**Pull request by rigazilla**
Minimal PR to hide empty string when logging a CR.

### CODE REVIEW

**Comment by dmvolod** [comment]
@rigazilla you should run `make codegen` to generate all CRD's and install bundle
I'm plan to remove `make crdgen` as it doesn't make sense

**Comment by rigazilla** [comment]
@dmvolod yeah right, though on 2.0.x branch build seems a little bit different. I've ran `make codegen install-bundle`

**Comment by dmvolod** [comment]
> @dmvolod yeah right, though on 2.0.x branch build seems a little bit different. I've ran `make codegen install-bundle`

Oh, yes, it doesn't use Operator SDK generator.

**Review by dmvolod — APPROVED** [review]

**Comment by rigazilla** [comment]
merged, thanks for the review @dmvolod !

File: pkg/apis/infinispan/v1/infinispan_types.go [hunk 1]
```diff
@@ -141,7 +141,7 @@ type InfinispanStatus struct {
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// +optional
 	Conditions              []InfinispanCondition `json:"conditions,omitempty"`
-	StatefulSetName         string                `json:"statefulSetName"`
+	StatefulSetName         string                `json:"statefulSetName,optional,omitempty"`
```
Comment (dmvolod) [inline pkg/apis/infinispan/v1/infinispan_types.go:144]: @rigazilla please pay attention, that this `StatefulSetName` properties not actual at master and should be ported.

**Pull request merged**